packages/plot/src/marks/RegressionMark.js

```diff
@@ -44,7 +44,7 @@ export class RegressionMark extends Mark {
   }
 
   queryResult(data) {
-    this.modelFit = toDataArray(data);
+    this.modelFit = toObjectArray(data);
```

Comment:
We could rewrite the logic to handle arrow but doesn't seem necessary since the data is so small.